pkg/handlers/app.go

```diff
@@ -20,6 +20,51 @@ import (
 	"github.com/replicatedhq/kots/pkg/version"
 )
 
+func (h *Handler) GetPendingApps(w http.ResponseWriter, r *http.Request) {
+	sess := session.ContextGetSession(r)
+	if sess == nil {
+		logger.Error(errors.New("invalid session"))
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	papp, err := store.GetStore().GetPendingAirgapUploadApp()
+	if err != nil {
+		if store.GetStore().IsNotFound(err) {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			logger.Error(errors.Wrap(err, "failed to get pending app"))
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		return
+	}
+
+	defaultRoles := rbac.DefaultRoles() // TODO (ethan): this should be set in the handler
```

Comment:
this is not needed here, it's used in `ListApps` to check access for each app separately, but here we're only returning 1 pending app and access control should already be handled in the session middleware